pyblish_lite/model.py

```diff
@@ -629,16 +630,17 @@ def filterAcceptsRow(self, source_row, source_parent):
         """Exclude items in `self.excludes`"""
         model = self.sourceModel()
         item = model.items[source_row]
-
         key = getattr(item, "filter", None)
         if key is not None:
             regex = self.filterRegExp()
             if regex.pattern():
                 match = regex.indexIn(key)
                 return False if match == -1 else True
-
         for role, values in self.includes.items():
             data = getattr(item, role, None)
+            if hasattr(data, '__iter__'):
+                return True if any(value in values for value in data) else False
```

Comment:
This method is being called hundreds of times per second, possibly on each draw call. Which would include just scrolling. I'm not noticing any performance problem at the moment, but if you can think of any other way to avoid a nested loop that would be ideal. If not, let's see how it goes.